### PR TITLE
[BAEL-4136] What is this: [Ljava.lang.Object;?

### PR DESCRIPTION
**Pull request by ricardomoreirab**

### CODE REVIEW

File: core-java-modules/core-java-io-conversions-2/src/test/java/com/baeldung/arraystostring/JavaArraysToStringUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class JavaArraysToStringUnitTest {
+
+        @Test public void givenInstanceOfArray_whenTryingToConvertToString_thenNameOfClassIsShown() {
```
Comment (rpvilao) [inline core-java-modules/core-java-io-conversions-2/src/test/java/com/baeldung/arraystostring/JavaArraysToStringUnitTest.java:15]: let's have the annotations in different lines than the method definition

File: core-java-modules/core-java-io-conversions-2/src/test/java/com/baeldung/arraystostring/JavaArraysToStringUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class JavaArraysToStringUnitTest {
```
Comment (rpvilao) [inline core-java-modules/core-java-io-conversions-2/src/test/java/com/baeldung/arraystostring/JavaArraysToStringUnitTest.java:13]: indent is 4 spaces

Comment (ricardomoreirab) [reply core-java-modules/core-java-io-conversions-2/src/test/java/com/baeldung/arraystostring/JavaArraysToStringUnitTest.java:13]: As I'm using IntelliJ, I had some trouble with the XML not being applied; I used the Eclipse Formatter Plugin, let me know if it's okay.

File: core-java-modules/core-java-io-conversions-2/src/test/java/com/baeldung/arraystostring/JavaArraysToStringUnitTest.java (hunk 1 above)
Comment (ricardomoreirab) [reply core-java-modules/core-java-io-conversions-2/src/test/java/com/baeldung/arraystostring/JavaArraysToStringUnitTest.java]: Let me know if this is good.

**Pull request merged**